src/main/java/com/algorand/algosdk/v2/client/algod/GetBlock.java

```diff
@@ -24,6 +24,14 @@ public GetBlock(Client client, Long round) {
         this.round = round;
     }
 
+    /**
+     * The round from which to fetch block information.
+     */
+    public GetBlock round(Long round) {
```

Comment:
Path parameters shouldn't be added as query parameters.

Reply:
:-) Good catch. 

:-( This is an existing bug. 

And now, all these files will lose a method, some after release... 

src/main/java/com/algorand/algosdk/v2/client/algod/AccountInformation.java
src/main/java/com/algorand/algosdk/v2/client/algod/GetApplicationByID.java
src/main/java/com/algorand/algosdk/v2/client/algod/GetAssetByID.java
src/main/java/com/algorand/algosdk/v2/client/algod/GetBlock.java
src/main/java/com/algorand/algosdk/v2/client/algod/GetPendingTransactionsByAddress.java
src/main/java/com/algorand/algosdk/v2/client/algod/PendingTransactionInformation.java
src/main/java/com/algorand/algosdk/v2/client/algod/WaitForBlock.java
src/main/java/com/algorand/algosdk/v2/client/indexer/LookupAccountByID.java
src/main/java/com/algorand/algosdk/v2/client/indexer/LookupAccountTransactions.java
src/main/java/com/algorand/algosdk/v2/client/indexer/LookupApplicationByID.java
src/main/java/com/algorand/algosdk/v2/client/indexer/LookupAssetBalances.java
src/main/java/com/algorand/algosdk/v2/client/indexer/LookupAssetByID.java
src/main/java/com/algorand/algosdk/v2/client/indexer/LookupAssetTransactions.java
src/main/java/com/algorand/algosdk/v2/client/indexer/LookupBlock.java

Reply:
I don't think thats true, otherwise the function wouldn't be green in this PR

Reply:
you are right :-)
I fixed it here.